authorize_fields.go

```diff
@@ -194,6 +194,28 @@ func (t Args) CommandArgs() string {
 	return strings.Join(args, " ")
 }
 
+// isLineEnding returns true if a is a valid line ending for tacacs authorization
+// payload
+func isLineEnding(a string) bool {
+	return a == "<cr>" || a == "<CR>"
```

Comment:
```suggestion
	return strings.ToLower(a) == "<cr>"
```

would capture other edge cases such as "<cR>" and "<Cr>".

Reply:
welcome back Joe !